community-content/pytorch_pre_built_images_deployment/pytorch-text-classification-vertex-ai-train-tune-deploy.ipynb

```diff
@@ -0,0 +1,4106 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.            %env GOOGLE_APPLICATION_CREDENTIALS ''
</pre><p>what about running <code>gcloud auth application-default</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/898/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
Same here. Leave it as is to match the other notebooks.